Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
```

Comment:
Remove this source. Having multiple sources causes a security warning in bundler.